src/module.api/loan.controller.ts

```diff
@@ -174,6 +184,57 @@ export class LoanController {
     return await this.vaultService.get(id)
   }
 
+  /**
+   * List vault auction history.
+   *
+   * @param {string} id vaultId
+   * @param {number} batch liquidation height
+   * @param {number} index batch index
+   * @param {PaginationQuery} query
+   * @return {Promise<ApiPagedResponse<VaultAuctionBatchHistory>>}
+   */
+  @Get('/vaults/:id/auctions/:batch/history/:index')
```

Comment:
```suggestion
  @Get('/vaults/:id/auctions/:height/history/:batch')
```

Should be this, less confusion